src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -20,6 +20,7 @@ public class CommandInfo
 
         private readonly CommandService _commandService;
         private readonly Func<ICommandContext, object[], IServiceProvider, CommandInfo, Task> _action;
+        internal readonly IReadOnlyDictionary<char,char> _quotationAliases;
```

Comment:
IMO this should not be stored on the command, but I'm not sure if we have the ability to pass the command service to the parser where ParseArgsAsync is called. Might be worth taking a look?

Reply:
I looked at this again, and found that I could use the similar behavior of `IgnoreExtraArgs`.

https://github.com/Chris-Johnston/Discord.Net/blob/46e9cf64f3d3635de872720c308e922b13ab8114/src/Discord.Net.Commands/Info/CommandInfo.cs#L124